Makefile

```diff
@@ -58,6 +58,9 @@ worker:
 test:
 	$(TEST_CMD) -s -v
 
+test_:
```

Comment:
Why is it `test_`? I'd have no idea when to run this one and when to run other `test` ones. If I wanted to run last failed ones only, then maybe `test-last-failed`? Maybe I misunderstood it, though.

Reply:
It was like that on purpose, to save typing :)
I made the explicit test_last_failed now, and test_ as an alias.